### ISSUE TITLE
null bytes in ams2 xml output fails ingest on AAPB

### ISSUE THREAD

**Issue by ekemeyer** [WGBH-MLA/mla-tickets#10]
# Details 
already-existing assets on AAPB with fully-correct transcripts now fail to exist and cannot be re-created because pbcore XML output from ams2 contains "null byte"  GUIDS are:
cpb-aacip-1bf681f951f
cpb-aacip-25c635318f3
cpb-aacip-3400a9098df
cpb-aacip-49d917518a7
cpb-aacip-4f19f347e36
cpb-aacip-609c461a350
cpb-aacip-795d56d49c6
cpb-aacip-7bd5c95a121
cpb-aacip-986ad8f4a95
cpb-aacip-9f92625ca25
cpb-aacip-c89f6d4baed
cpb-aacip-ddac757b396
cpb-aacip-e5505dc8d8d
cpb-aacip-f8013794a56

Submitted by: Miranda
Priority: High (within the week)
URL: example XML: https://ams2.wgbh-mla.org/concern/asset_resources/cpb-aacip-1bf681f951f.xml
Slack message thread:

**Comment by foo4thought** [comment]
I downloaded https://ams2.wgbh-mla.org/concern/asset_resources/cpb-aacip-1bf681f951f.xml ; after inspection in BBEdit for any offending "null byte",  it reports none.
I searched the AAPB2 repo for any combination of the error string 'string contains null byte' and yielded nothing.  
grok'd the repo's scripts/lib/cleaner.rb to look for anything reducing text.
OMG
I found the problem:  it's a DOCX original that got uploaded to S3 as a TXT.
